src/main/java/com/amashchenko/maven/plugin/gitflow/AbstractGitFlowMojo.java

```diff
@@ -316,10 +321,18 @@ private String getCurrentProjectOutputTimestamp() throws MojoFailureException {
      * @throws MojoFailureException
      *             If project loading fails.
      */
-    private MavenProject reloadProject(MavenProject project) throws MojoFailureException {
+    private MavenProject reloadProject(final MavenProject project) throws MojoFailureException {
         try {
-            ProjectBuildingResult result = projectBuilder.build(project.getFile(), mavenSession.getProjectBuildingRequest());
-            return result.getProject();
+            List<ProjectBuildingResult> result = projectBuilder.build(
+            		Collections.singletonList(project.getFile()), true, mavenSession.getProjectBuildingRequest());
+            
+            for (ProjectBuildingResult projectBuildingResult : result) {
+				MavenProject resultProject = projectBuildingResult.getProject();
+				if (!Objects.equals(resultProject.getGroupId(), project.getGroupId())) continue;
+				if (!Objects.equals(resultProject.getArtifactId(), project.getArtifactId())) continue;
+				return resultProject;
+			}
```

Comment:
Looks like the project uses spaces for indentation, whereas this part mixes spaces and tabs.

```suggestion
                    Collections.singletonList(project.getFile()), true, mavenSession.getProjectBuildingRequest());

            for (ProjectBuildingResult projectBuildingResult : result) {
                MavenProject resultProject = projectBuildingResult.getProject();
                if (!Objects.equals(resultProject.getGroupId(), project.getGroupId())) continue;
                if (!Objects.equals(resultProject.getArtifactId(), project.getArtifactId())) continue;
                return resultProject;
            }
```

Reply:
Thank you @beatngu13 for your corrections.
I have incorporated those changes in my first commit.